config/ose-placeholders.js

```diff
@@ -14,15 +14,15 @@ var placeholders = {
   "mongo.admin_auth.pass": "{{env.MONGODB_ADMIN_PASSWORD}}",
   "fhredis.host": "{{env.REDIS_SERVICE_SERVICE_HOST}}",
   "fhredis.port": "{{env.REDIS_SERVICE_SERVICE_PORT}}",
-  "fhmetrics.host": "fh-metrics-service",
+  "fhmetrics.host": "fh-metrics-service.{{env.MBAAS_NAMESPACE}}",
```

Comment:
ah right, sorry...I saw this the other way around: I thought it was being removed as part of this change.